src/hooks/useTransactor.ts

```diff
@@ -48,6 +62,38 @@ function logTx({
   )
 }
 
+function sendKeypTransaction({
+  contract,
+  functionName,
+  args,
+  accessToken,
+  value,
+}: {
+  contract: Contract
+  functionName: string
+  args: string[]
+  accessToken: string
+  value?: BigNumberish
+}) {
+  const { address } = contract
+
+  // console.log(
+  //   'asdf req data',
+  //   { address },
+  //   { abi },
+  //   { accessToken },
+  //   contract.interface.fragments,
+  // )
+
+  return writeContract({
+    accessToken,
+    address,
+    abi: abiFunctionName(functionName, contract.interface),
```

Comment:
we need to send the fn signature (i.e. `Pay(uint256, uint256, ...)`) instead of the full abi